docs/releases/updating_ez_platform.md

```diff
@@ -120,6 +120,10 @@ On PHP conflict | 16.02 and later requires PHP 5.5 or higher
 
 Because from release 16.02 onwards eZ Platform is compatible only with PHP 5.5 and higher, the update command above will fail if you use an older PHP version. Please update PHP to proceed.
 
+!!! Common errors
+
+If you experienced issues during the update, please check [Common errors](https://doc.ezplatform.com/en/master/getting_started/about_composer/#cloning-failed-using-an-ssh-key) section.
```

Comment:
This sort of note box requires the box body to be indented by 4 spaces.
Also, you can use a relative link which will work for all versions: `(../getting_started/about_composer/#cloning-failed-using-an-ssh-key)`